contents/docs/tutorials/retention.md

```diff
@@ -0,0 +1,174 @@
+
+However, this does not mean retention on the core offering is not an important metric for these businesses. In fact, it is arguably more important. Consider phone manufacturers, for example, who need to look at retention over years (with respect to the main product: the phones themselves). If such businesses are unable to retain users, they lose customers for years, and potentially even forever. Once a user decides to switch brands for their new phone, it is much harder to get them back. 
+
+##### One-Off Products and Services
+
+**Example:** Real Estate Brokers
+
+Finally, on the very end of the spectrum, there are certain businesses that should focus more on other metrics like referrals instead of retention.
+
+Take a real estate broker that only does sales (no rentals), for example. Once the customer purchases a property, it is unlikely you will have them as a customer again any time soon. Rather, you want that customer to be happy with the service and recommend it to others. 
+
+There is still some retention tracking that can be done, however. Let's say you have an online platform with listed properties. While you are not too interested in retention post-purchase, you can (and should) still measure retention on the platform pre-purchase. How often are users coming into your platform to check for listings? Do you appear to be their top choice for searches? After they use the platform for the first time, do they come back?
+
+#### Tracking Retention Metrics in PostHog
+
+Having determined how to measure retention in a way that is relevant to your business, you then need to ensure those goals are reflected in your analytics processes within PostHog. 
+
+##### Key Concepts
+
+Before we proceed to walkthroughs of PostHog's functionality, you should be aware that PostHog offers two different retention metrics that you may want to use. These are:
+
+[**Retention**](/docs/features/retention)
+
+Retention tracks repeated usage of your product over set time periods. For example, you can look at retention for a given event over the past week, visualizing how retention changed on each specific day of that week. Retention can be found as a tab in 'Insights'.
+
+[**Stickiness**](/docs/features/trends/#trend-segmentation-by-stickiness)
+
+Stickiness is also a metric of repeated usage, but it disregards the absolute time period in favor of a relative metric. In practical terms, while retention will give you an answer for how many users visited your page on October 22nd and then visited again on October 23rd, stickiness doesn't care about the dates, rather offering an answer for how many users came back to your site on day 2 after visiting on day 1 (irrespective of what the exact dates were). Tracking stickiness in PostHog is done in the 'Trends' tab in 'Insights'.
```

Comment:
I'd add that stickiness makes more sense when you are *very* early stage and do not have much data.

ie Since retention splits everything out by month etc, then if you have ie 5 users, it's a bit meaningless to break that data up further.